CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: Using limit=0 and options=count to get only the count of elements (#1492)
```

Comment:
```suggestion
- Fix: allow limit=0 in all paginated operations in the NGSIv2 API (entities, entity types, subscriptions and registrations)  (#1492)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/64cf793d7fae599dcef70fc063ce6302eeb9e5bd